docs/usage/updates-strategy.md

```diff
@@ -101,3 +101,35 @@ Reboot windows can be separately configured in multiple snippets, as long as eac
  * `length_minutes`: non-zero window duration, in minutes
 
 For convenience, multiple entries can be defined with overlapping times, and each window definition is allowed to cross day and week boundaries (wrapping to the next day).
+
+To configure a non-UTC time zone for reboot windows, specify the `time_zone` field in a `updates.periodic` entry. The specified time zone must be either `"localtime"` or a time zone name from the [IANA Time Zone Database][IANA_tz_db] (you can find an unofficial list of time zone names [here][wikipedia_tz_names]). 
+If using `"localtime"`, `/etc/localtime` must either be a symlink to a valid `tzfile` entry in your system's local time zone database (under `/usr/share/zoneinfo/`), or not exist, in which case `UTC` is used.
+
+Note that you can only specify a single time zone for _all_ reboot windows.
+
+:warning: **When using non UTC time zones:** reboot window clock times are always obeyed, therefore **reboot window lengths may vary**!
+Reboot windows may be lengthened or shortened due to shifts in clock time. For example, with the `US/Eastern` time zone, on "fall back" day, a specified reboot window may be lengthened by up to one hour; on "spring forward" day, a specified reboot window may be shortened by up to one hour, or skipped entirely.
+
+Example of periodic reboot windows using the `US/Eastern` time zone.
```

Comment:
It may better to start with a simpler non-DST example first, and move this longer `US/Eastern` example and explanation to its own `## Timezone caveats` section.
`America/Panama` should be a good enough pick as it never introduced DST (so far) and is fairly recognizable by the average user.